src/mmw/sass/base/_map.scss

```diff
@@ -24,6 +24,15 @@
 .leaflet-draw-toolbar a {
 	background-image: url('../images/spritesheet.png');
 }
+
 .leaflet-retina .leaflet-draw-toolbar a {
 	background-image: url('../images/spritesheet-2x.png');
 }
+
+.leaflet-control-layers-toggle{
+  background-image: url('/static/images/layers.png');
```

Comment:
Consider making a whitespace commit to fix the different indentation patterns in this file.